Frameworks/CoreGraphics/CGContext.mm

```diff
@@ -493,17 +493,17 @@ inline bool ShouldDraw() {
         return SUCCEEDED(_firstErrorHr) && CurrentGState().ShouldDraw();
     }
 
-    inline void PushBeginDraw() {
-        if ((_beginEndDrawDepth)++ == 0) {
+    inline void PushBeginDraw(bool forceBeginDraw = false) {
+        if ((_beginEndDrawDepth)++ == 0 || forceBeginDraw) {
```

Comment:
@DHowett-MSFT out of curiosity, what would be your proposed alternative to the override flag?